sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/GenericRecordReadConverter.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.sdk.extensions.sql.meta.provider.parquet;
+package org.apache.beam.sdk.io.parquet;
 
 import com.google.auto.value.AutoValue;
 import java.io.Serializable;
```

Comment:
Can you make this class package-private? Users may be more tempted to use it now that it's outside of the SQL extensions.

This could be a generally useful transform, we may want to move it outside of the parquet package and make it public... but let's not do that now.